src/DataWrangling/restoring.jl

```diff
@@ -1,8 +1,9 @@
 using Oceananigans: location
 using Oceananigans.Grids: AbstractGrid, node, on_architecture
 using Oceananigans.Fields: interpolate!, interpolate, location, instantiated_location
-using Oceananigans.OutputReaders: Cyclical, TotallyInMemory, AbstractInMemoryBackend, FlavorOfFTS, time_indices
+using Oceananigans.OutputReaders: Cyclical, TotallyInMemory, AbstractInMemoryBackend, FlavorOfFTS, time_indices, FieldTimeSeries
```

Comment:
why `using Oceananigans.OutputReaders: FieldTimeSeries` and not `import`? Don't we want to add a method?

Reply:
Hm, you are right, but I'm not sure why it is working... Either way, I have moved it to the import line...